app/js/thisissoon-core/filters/linkDisplay.js

```diff
@@ -9,7 +9,7 @@ angular.module("thisissoon.core").filter("linkDisplay", [
     "$filter",
     function($filter) {
         return function(input) {
-            return input.replace("http://", "").replace("www.", "");
+            return input.replace("http://", "").replace("https://", "").replace("www.", "");
         };
```

Comment:
@jamesjwarren would it not be better to use regex for this? something like: `/[a-zA-Z]+:\/\//` would match any protocol and: `/w{3}./`would match `www.`